apps/condo/domains/user/schema/CheckUserExistenceService.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Generated by `createservice user.CheckUserExistenceService --type queries`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND, WRONG_FORMAT, DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/user/access/CheckUserExistenceService')
+const { ConfirmPhoneAction } = require('@condo/domains/user/utils/serverSchema')
+const { checkDayAnonymousRequestLimitCounters } = require('@condo/domains/user/utils/serverSchema/requestLimitHelpers')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    MULTIPLE_USERS_FOUND: {
+        query: 'checkUserExistence',
+        code: NOT_FOUND,
+        type: 'MULTIPLE_USERS_FOUND',
+        message: 'Multiple users found',
+    },
+    INVALID_PHONE_NUMBER: {
+        query: 'checkUserExistence',
+        code: BAD_USER_INPUT,
+        type: 'INVALID_PHONE_NUMBER',
+        message: 'Invalid phone number',
+    },
+    TOKEN_NOT_FOUND: {
+        query: 'checkUserExistence',
+        variable: ['data', 'token'],
+        code: NOT_FOUND,
+        type: 'TOKEN_NOT_FOUND',
+        message: 'Token not found',
+    },
+    DV_VERSION_MISMATCH: {
+        query: 'checkUserExistence',
+        variable: ['data', 'dv'],
+        code: BAD_USER_INPUT,
+        type: DV_VERSION_MISMATCH,
+        message: 'Wrong value for data version number',
+    },
+    WRONG_SENDER_FORMAT: {
+        query: 'checkUserExistence',
+        variable: ['data', 'sender'],
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Invalid format of "sender" field value',
+        correctExample: '{ dv: 1, fingerprint: \'example-fingerprint-alphanumeric-value\'}',
+    },
+}
+
+const CheckUserExistenceService = new GQLCustomSchema('CheckUserExistenceService', {
+    types: [
+        {
+            access: true,
+            type: 'input CheckUserExistenceInput { dv: Int!, sender: SenderFieldInput!, confirmActionToken: ID!, userType: UserTypeType! }',
```

Comment:
What about service users? You block them in other mutations

Reply:
Not all mutations are blocked for service users. So I decided to allow this mutation for them